packages/tupaia-web/src/api/mutations/useLogin.ts

```diff
@@ -21,6 +21,7 @@ export const useLogin = () => {
 
   const { closeModal, navigateToModal } = useModal();
 
+  gaEvent('User', 'Log in', 'Attempt');
```

Comment:
I think we should put this inside an `onMutate` event, similar to the `onSuccess`, because otherwise this will get called more often than a login attempt (as the mutation call isn't called just whenever the hook is used, like `useQuery` is)